delta/introduction.adoc

```diff
@@ -27,7 +40,7 @@ The client receives only events on nodes within that scope.
 
 A typical workflow for a client would be:
 
-1. Client <<qry-SignOn, connects>> to repository and starts a participation.
+1. Client <<qry-SignOn, connects>> to the server and starts a participation.
```

Comment:
this contradicts line 17

Reply:
with a server, for a specific repository